tools/online_editor/cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
is this file necessary?

Reply:
It won't be there next time I push this :-)